cross-modal-search/README.md

```diff
@@ -46,11 +34,11 @@ for them to have different modalities.
 the body text and the title are from the same mime type (text),
 but can be considered of different modalities (distributions).
```

Comment:
Can you be more descriptive on what distributions mean in this case?

Reply:
5f33ee0ac0a037bf57c8c44113a7548b79a90bf5